app/uk/gov/hmrc/tariffclassificationfrontend/views/refer_case.scala.html

```diff
@@ -19,35 +19,78 @@
 @import uk.gov.hmrc.tariffclassificationfrontend.models.Case
 @import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
 @import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.{case_heading, contact_details}
-@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.components.input_yes_no
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.components.{confirm_or_cancel_buttons, warning}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.components.{input_textarea, input_radiogroup, input_checkbox_group, input_text}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.statuses.status_change_upload
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{CaseReferral, ReferralReason}
+@import uk.gov.hmrc.tariffclassificationfrontend.models.ReferralReason._
+@import uk.gov.hmrc.tariffclassificationfrontend.views.forms.components.{CheckOption, RadioOption}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.error_summary
+@(c: Case, form: Form[CaseReferral])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.error_summary
-@(c: Case, form: Form[Boolean])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@inputTextContent() = {
+        @input_text(label = "Give details of who you are referring this case to", field = form("referManually"), inputClass = Some("w-50"),
+        customGroupClasses = Some("panel panel-border-narrow js-hidden pl-30"), customGroupId = Some("other-detail"))
```

Comment:
Shouldnt there just be a div inside the radio group with all the `panel` stuff given its on every one?

Reply:
ok think got a solution for this, that requires less specification of stuff in client class that uses the input radio group